src/library/Uno.Material/Styles/Controls/BottomNavigationBarItem.xaml

```diff
@@ -0,0 +1,152 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:controls="using:Uno.Material.Controls"
+					xmlns:converters="using:Uno.Material.Converters">
+
+	<StaticResource x:Key="BottomNavLabelFocusColorBrush"
+					ResourceKey="OnPrimaryBrush" />
+	<x:Double x:Key="BottomNavLabelFontSize">12</x:Double>
+	<StaticResource x:Key="BottomNavIconFocusColorBrush"
+					ResourceKey="OnPrimaryBrush" />
+	<StaticResource x:Key="BottomNavLabelColorBrush"
+					ResourceKey="OnPrimaryMediumBrush" />
+	<StaticResource x:Key="BottomNavIconColorBrush"
+					ResourceKey="OnPrimaryMediumBrush" />
+	<StaticResource x:Key="BottomNavNotifBackgroundColorBrush"
+					ResourceKey="ErrorBrush" />
+	<StaticResource x:Key="BottomNavNotifLabelColorBrush"
+					ResourceKey="OnErrorBrush" />
+
+	<converters:FromEmptyStringToValueConverter x:Name="EmptyToCollapsed"
+												NotNullOrEmptyValue="Visible"
+												NullOrEmptyValue="Collapsed" />
+
+	<CubicEase x:Key="EaseOutFunction"
+			   EasingMode="EaseOut" />
+
+	<Style x:Key="DefaultBottomNavigationBarItemStyle"
+		   TargetType="controls:BottomNavigationBarItem">
+		<Setter Property="Foreground"
+				Value="{StaticResource BottomNavLabelColorBrush}" />
+		<Setter Property="Margin"
+				Value="0,4" />
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="controls:BottomNavigationBarItem">
+					<Grid Margin="{TemplateBinding Margin}"
+						  Background="Transparent">
+						<VisualStateManager.VisualStateGroups>
+							<VisualStateGroup x:Name="CommonStates">
+								<VisualStateGroup.Transitions>
+									<VisualTransition x:Name="ToPressedTransition"
+													  To="Pressed"
+													  GeneratedDuration="0">
+										<Storyboard>
+											<DoubleAnimation Storyboard.TargetName="Scale"
```

Comment:
"Scale" is misleading. I'd use RippleScale so that we don't think it's the actual button that we scale.

Reply:
sounds good 👍